ceph/ceph.py

```diff
@@ -1846,7 +1846,9 @@ def install_ceph_ansible(self, rhbuild, **kw):
                 raise CommandFailed(err)
 
         if kw.get('upgrade'):
-            self.exec_command(sudo=True, cmd='yum update metadata')
+            self.exec_command(sudo=True,
+                              cmd='yum update meta',
```

Comment:
@sunilkumarn417  can you add ec_check=False to  all references of 'yum update metadata' in the repo or remove it completely ..afaik its deprecated functionality and a very old way to update yum cache in rhel6..we dont need to manually update metadata in rhel7 i believe ..every time we use yum it searches for latest repos added and picks latest pkgs

Reply:
https://stackoverflow.com/questions/13328283/equivalent-in-yum-of-apt-get-update